youtube_dl/extractor/fujitv.py

```diff
@@ -1,35 +0,0 @@
-# coding: utf-8
-from __future__ import unicode_literals
-
-from .common import InfoExtractor
-
-
-class FujiTVFODPlus7IE(InfoExtractor):
-    _VALID_URL = r'https?://i\.fod\.fujitv\.co\.jp/plus7/web/[0-9a-z]{4}/(?P<id>[0-9a-z]+)'
```

Comment:
Do not remove. This still can be used directly.

Reply:
Okay, reverted it back. But it seems returning 403 for all pages I think